packages/pluggableWidgets/rating-web/src/components/Rating.tsx

```diff
@@ -0,0 +1,109 @@
+              } else {
+                  props.onChange?.(currentIndex);
+              }
+          }
+        : undefined;
+
+    const focusItem = (direction: Direction): void => {
+        if (containerRef.current) {
+            const currentFocusedElement = containerRef.current.querySelector(".rating-item:focus");
+            if (direction === Direction.NEXT && currentFocusedElement?.nextSibling) {
+                (currentFocusedElement.nextSibling as HTMLDivElement).focus();
+            } else if (direction === Direction.PREVIOUS && currentFocusedElement?.previousSibling) {
+                (currentFocusedElement.previousSibling as HTMLDivElement).focus();
+            }
+        }
+    };
+
+    return (
+        <div
+            className={classNames("widget-rating", props.className)}
+            data-focusindex={props.tabIndex ?? 0}
+            role="radiogroup"
+            style={props.style}
+            onKeyDown={event => {
+                switch (event.key) {
+                    case "Left": // Microsoft Edge value
+                    case "ArrowLeft":
+                        event.preventDefault();
+                        focusItem(Direction.PREVIOUS);
+                        break;
+                    case "Right": // Microsoft Edge value
+                    case "ArrowRight":
+                        event.preventDefault();
+                        focusItem(Direction.NEXT);
+                        break;
+                }
+            }}
+            ref={containerRef}
+        >
+            {Array.from({ length: props.maximumValue }, (_, index) => {
+                const currentIndex = index + 1;
+                return (
+                    <div
+                        aria-checked={currentIndex === props.value}
+                        aria-label={currentIndex.toString()}
+                        className={classNames("rating-item", { disabled: props.disabled })}
+                        key={`icon_${currentIndex}`}
+                        onClick={() => {
+                            setHover(undefined);
+                            onClickAction?.(currentIndex);
+                        }}
+                        onKeyDown={event => {
+                            if (event.key === " " || event.key === "Enter") {
+                                event.preventDefault();
+                                onClickAction?.(currentIndex);
+                            }
+                        }}
+                        onMouseOut={() => {
+                            setHover(undefined);
+                        }}
+                        onMouseOver={() => {
+                            if (!props.disabled && props.animated) {
+                                setHover(currentIndex);
+                            }
+                        }}
+                        role="radio"
+                        tabIndex={index === (props.value > 0 ? props.value - 1 : 0) ? 0 : -1}
```

Comment:
Nit: I feel like this is slightly more readable, but i'll leave it up to u
```suggestion
                        tabIndex={index === Math.max(props.value - 1, 0) ? 0 : -1}
```